patterns/core/sass/_plugins.scss

```diff
@@ -3,3 +3,5 @@
 //////////////////////////////
 @import '../../../bower_components/sass-toolkit/stylesheets/toolkit';
 @import '../../../bower_components/ibm-colors/ibm-colors';
```

Comment:
:( wish we could just write a mixin and have a list of bower imports. I suppose for now its just wishful thinking